students/V_Medina/lesson_02/Grid_Printer_2.py

```diff
@@ -0,0 +1,21 @@
+'''
+Victor A Medina
+Assignment 2: Grid Printer Part 2
+'''
+def print_grid(n):
+	plus_row=''
```

Comment:
Make sure you have your editor set up to use spaces instead of tabs.  Indentations should be 4 spaces in Python